model/patch/patch.go

```diff
@@ -973,3 +973,33 @@ func (p PatchesByCreateTime) Less(i, j int) bool {
 func (p PatchesByCreateTime) Swap(i, j int) {
 	p[i], p[j] = p[j], p[i]
 }
+
+type PatchesByStatus []Patch
+
+func (p PatchesByStatus) Len() int {
+	return len(p)
+}
+
+func (p PatchesByStatus) Less(i, j int) bool {
+	return p[i].patchStatusPriority() < p[j].patchStatusPriority()
+}
+
+func (p PatchesByStatus) Swap(i, j int) {
+	p[i], p[j] = p[j], p[i]
+}
+
+// patchStatusPriority answers the question of what the patch status should be
+// when the patch status and the status of it's children are different
+func (p *Patch) patchStatusPriority() int {
+	switch p.Status {
+	case evergreen.PatchCreated:
+		return 10
+	case evergreen.PatchStarted:
```

Comment:
Actually shouldn't started take precedent over created? If anything is currently running I would imagine that we should reflect that.

Reply:
I'm concerned about the following presumably very common scenario: 
A patch has a child patch that's set to start when the parent finishes. The user sees the patch has "started", because the parent started. The parent succeeds and the child will start soon, but hasn't yet. The child patch's "created" status now takes precedent. So the user sees the patch go from started to created, which is probably confusing and feels like a mistake.

Reply:
Gotcha; is it possible we shouldn't be using sort then? I find it just as confusing to say "Created" when things have already run. Instead of sorting, could we do like a "getStatusForPatches" that returns Started if any patches have succeeded/failed but there are still started/created tasks? This gives us more flexibility.